src/ansys/dynamicreporting/core/adr_service.py

```diff
@@ -220,39 +221,85 @@ def __init__(
                 self.logger.error(f"Error starting the Docker Container.\n{str(e)}\n")
                 raise e
 
-        elif ansys_installation:  # pragma: no cover
-            # verify path
-            if not os.path.isdir(ansys_installation):
-                raise InvalidAnsysPath(ansys_installation)
-
-        if ansys_installation != "docker" and ansys_installation is not None:  # pragma: no cover
-            # Not using docker
-            # Backward compatibility: if the path passed is only up to the version directory,
-            # append the CEI directory
-            if ansys_installation.endswith("CEI") is False:
-                ansys_installation = os.path.join(ansys_installation, "CEI")
-                self._ansys_installation = ansys_installation
-                # verify new path
-                if not os.path.isdir(ansys_installation):
-                    # Option for local development build
-                    if os.environ.get("CEIDEVROOTDOS") is not None:
-                        self._ansys_installation = os.environ.get("CEIDEVROOTDOS")
-                    else:
-                        raise InvalidAnsysPath(ansys_installation)
-            if self._ansys_version is None:
-                # try to get version from install path
-                matches = re.search(r".*v([0-9]{3}).*", self._ansys_installation)
-                if matches is None:
-                    # Option for local development build
-                    if os.environ.get("ANSYS_REL_INT_I") is not None:
-                        self._ansys_version = int(os.environ.get("ANSYS_REL_INT_I"))
+        else:  # pragma: no cover
+            # Not using docker. Make a list of directory names to consider:

```

Comment:
Check these locations for distros.   User-passed names first, if 'enve' can be loaded, try that (easy way to make a local build work via 'cpython'), otherwise try AWP_ROOT environmental variables and finally CEIDEVROOTDOS.